docs/concepts/policy/pod-security-policy.md

```diff
@@ -125,141 +399,153 @@ to the volume sources that are defined when creating a volume:
+- *RunAsAny* - No default provided. Allows any `supplementalGroups` to be
+specified.
 
-To modify policy interactively, use `kubectl edit`:
+### Privilege Escalation
+
+These options control the `AllowPrivilegeEscalation` container option. This bool
+directly controls whether the
+[`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt)
+flag gets set on the container process. This flag will prevent `setuid` binaries
+from changing the effective user ID, and prevent files from enabling extra
+capabilities (e.g. it will prevent the use of the `ping` tool). This behavior is
+required to effectively enforce `MustRunAsNonRoot`.
+
+It defaults to `nil`. The default behavior of `nil` allows privilege escalation
+so as to not break setuid binaries. Setting it to `false` ensures that no child
+process of a container can gain more privileges than its parent.
+
+**AllowPrivilegeEscalation** - Gates whether or not a user is allowed to set the
+security context of a container to `allowPrivilegeEscalation=true`. This
+defaults to allowed.
+
+**DefaultAllowPrivilegeEscalation** - Sets the default for the
+`AllowPrivilegeEscalation` option. The default behavior is to allow privilege
+escalation so as to not break setuid binaries.
+
+### Capabilities
+
+Linux capabilities provide a finer grained breakdown of the privileges
+traditionally associated with the superuser. Some of these capabilities can be
+used to escalate privileges or for container breakout, and may be restricted by
+the PodSecurityPolicy. For more details on Linux capabilities, see
+[Capabilities(7)](http://man7.org/linux/man-pages/man7/capabilities.7.html).
+
+The following fields take a list of capabilities, specified as the capability
+name in ALL_CAPS without the `CAP_` prefix.
+
+**AllowedCapabilities** - Provides a whitelist of capabilities that may be added
+to a container. The default set of capabilities are implicitly allowed. The
+empty set means that no additional capabilities may be added beyond the default
+set.
+
+**RequiredDropCapabilities** - The capabilities which must be dropped from
+containers. These capabilities are removed from the default set, and must not be
+added. Capabilities listed in `RequiredDropCapabilities` must not be included in
+`AllowedCapabilities` or `DefaultAddCapabilities`.
+
+**DefaultAddCapabilities** - The capabilities which are added to containers by
+default, in addition to the runtime defaults. The Docker runtime defaults are:
+
+- `AUDIT_WRITE`
+- `CHOWN`
+- `DAC_OVERRIDE`
+- `FOWNER`
+- `FSETID`
+- `KILL`
+- `MKNOD`
+- `NET_BIND_SERVICE`
+- `NET_RAW`
+- `SETFCAP`
+- `SETGID`
+- `SETPCAP`
+- `SETUID`
+- `SYS_CHROOT`
 
-```shell
-$ kubectl edit psp permissive
-```
+### SELinux
 
-This command will open a default text editor where you will be able to modify policy.
+- *MustRunAs* - Requires `seLinuxOptions` to be configured if not using
+pre-allocated values. Uses `seLinuxOptions` as the default. Validates against
```

Comment:
AFAIK there is no "pre-allocated values" and this sentence is wrong.

Reply:
I'm actually not familiar with SELinux support at all. This was copied from the previous version of this page. Can you suggest a replacement?

Reply:
I suggest the following:
```
*MustRunAs* - Requires `seLinuxOptions` to be configured. Uses `seLinuxOptions` as the default and as the value to validate against.
```

@pweil- @liggitt Please, confirm that there is no "pre-allocated values" in Kubernetes.